distro/run/assembly/resources/start.sh

```diff
@@ -4,8 +4,34 @@ BASEDIR=$(dirname "$0")
 runScript=$BASEDIR/internal/run.sh
 
 if [ $# -eq 0 ]; then
+
+  # open a browser (must be done first)
+  UNAME=`which uname`
+  if [ -n "$UNAME" -a "`$UNAME`" = "Darwin" ]
+  then
+    BROWSERS="open"
+  else
+    BROWSERS="xdg-open gnome-www-browser x-www-browser firefox chromium chromium-browser google-chrome"
+  fi
+
+  if [ -z "$BROWSER" ]; then
+    for executable in $BROWSERS; do
+      BROWSER=`which $executable 2> /dev/null`
+      if [ -n "$BROWSER" ]; then
+        break;
+      fi
+    done
+  fi
+
+  if [ -z "$BROWSER" ]; then
+    (sleep 5; echo -e "We are sorry... We tried all we could do but we couldn't locate your default browser... \nIf you want to see our default website please open your browser and insert this URL:\nhttp://localhost:8080/camunda-welcome/index.html";) &
+  else
+    (sleep 10; $BROWSER "http://localhost:8080/";) &
+  fi
+
```

Comment:
Do we have to do that first in shell? I believe Run is started in the background and you can open a browser afterward, can't you?

Reply:
For some reason, if I change the order, the browser isn't opened.

Reply:
That's weird. Well, if it works like that, I'm good 👍